Resources/Prototypes/EntityLists/pmm_weapons.yml

```
@@ -0,0 +1,16 @@
+    - WeaponRifleM90GrenadeLauncher
+    - WeaponRifleLecter
+    - WeaponSubMachineGunC20r
+    - WeaponSubMachineGunDrozd
+    - WeaponSubMachineGunVector
+    - WeaponSubMachineGunWt550
+    - WeaponShotgunBulldog
+    - WeaponShotgunEnforcer
+    - WeaponRevolverDeckard
+    - WeaponRevolverInspector
+    - WeaponRevolverPython
+    - WeaponLightMachineGunL6
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)